src/Storefront/Resources/app/storefront/src/plugin/cookie/cookie-configuration.plugin.js

```diff
@@ -425,21 +427,13 @@ export default class CookieConfiguration extends Plugin {
         this.closeOffCanvas();
     }
 
-    acceptAllCookies() {
```

Comment:
Before removing a public method, it needs to be deprecated. What do you think of keeping this one, but adding an additional, optional parameter for controlling the behaviour as needed?